privacyidea/api/lib/prepolicy.py

```diff
@@ -1254,7 +1254,7 @@ def pushtoken_add_config(request, action):
     This is a token specific wrapper for push token for the endpoint
     /token/init
     According to the policy scope=SCOPE.ENROLL,
-    action=TTL or action=REGISTRATION_URL the parameters are added to the
+    action=SSL_VERIFY_NO or action=FIREBASE_CONFIG the parameters are added to the
```

Comment:
Did You mean just `SSL_VERIFY`?

Reply:
Yes, this is a remnant of my first bool try.
I will fix this.

Reply:
done in 2637050c